CMSgov/design-system
# [WNMGDS-1245] Update autocomplete button

## Summary

[Jira ticket](https://jira.cms.gov/browse/WNMGDS-1245)

- Added `ds-u-padding-right--0` to the Autocomplete clear button
  - I attempted updating padding on the `ds-c-autocomplete__clear-btn` class, however the cascade of Buttons loading after Autocomplete caused this style to be overwritten without adding `!important` or overly specific classes to this component
  - Using a utility class in this context poses little to no risk as we don't offer users a prop to add their own CSS class to this button - therefore it won't interfere with user implementation
- Updated Autocomplete stories to comply with sentence casing rules
- Updated snapshot tests to match new markup changes

## How to test

1. Visit [Autocomplete docs](https://cmsgov.github.io/design-system/branch/WNMGDS-1245/bugfix-autocomplete-btn-alignment/components/autocomplete/?theme=core)
2. See that clear button is right aligned with input field

### If this is a change to code:


### If this is a change to documentation:
